api_core/google/api_core/protobuf_helpers.py

```diff
@@ -247,3 +249,42 @@ def setdefault(msg_or_dict, key, value):
     """
     if not get(msg_or_dict, key, default=None):
         set(msg_or_dict, key, value)
+
+
+def fieldmask(original, modified):
+    """Constructs a field mask from two proto messages.
+
+    Args:
+        original (~google.protobuf.message.Message): the original message. 
+        modified (~google.protobuf.message.Message): the modified message.
+
+    Returns:
+        FieldMask: returns a FieldMask object representing the differences
+            between made from the original message to the modified message.
+
+    Raises:
+        ValueError: If the ``original`` or ``modified`` are not subclasses of
+        ``google.protobuf.message.Message``.
+    """
+    if not isinstance(original, Message) or not isinstance(modified, Message):
+        raise ValueError('The parameters passed must be a subclass of google.protobuf.message.Message.')
+    if type(original) != type(modified):
+        raise ValueError('The parameters passed must be of the same type.')
+    answer = []
+    seen = [] 
+    for field, _ in original.ListFields():
```

Comment:
nit: Please put a blank newline above control statements, it tends to make code slightly more readable.

Reply:
Done!